.github/workflows/linux_job.yml

```
@@ -152,6 +153,20 @@ jobs:
           path: ${{ inputs.repository || github.repository }}
           fetch-depth: ${{ inputs.fetch-depth }}
 
+      - name: Calculate docker image
+        id: calculate-docker-image
+        uses: huydhn/test-infra/.github/actions/calculate-docker-image@calculate-docker-image
```

```suggestion
        uses: ./test-infra/.github/actions/calculate-docker-image
```

You can actually just reference the local action since the actual test job will check out your local version of this and this job in particular is guaranteed to have a local check out of test-infra (at all times)